### DISCUSSION TITLE
Vote for Smart Connections as Obsidian's Best New Plugin

### DISCUSSION THREAD

**Discussion by brianpetro** [brianpetro/obsidian-smart-connections#410] [Announcements]
If you haven't voted yet, right now you can really make a difference!

Smart Connections is currently in fourth place, and I'd love to land in the Top 3 🏆

We just need ~10 more votes. So if you haven't already voted, right now you really can make a difference.

Show your support by sending a screenshot of your vote to brian@smartconnections.app and I'll send you instructions for getting early access to V2.0! 

Vote here:
https://discord.com/channels/686053708261228577/702717892533157999/1193993954346139729

Note: The link above only works if you are already logged in to the Discord server.

Link to join the discord server: https://discord.gg/obsidianmd

If you're not already in the Obsidian discord, voting requires joining the Obsidian Discord server, going to the 'Announcements' section and clicking the 'S' to vote for Smart Connections in the "Best New Plugin" category.

Thanks for your support 🌴 🤓

![Screenshot_20240112-114240](https://github.com/brianpetro/obsidian-smart-connections/assets/1886014/946bda74-114c-469f-8307-c7fd5949dd95)

**Comment by brianpetro** [comment]
6 more to tie for third 🥉🤞

**Reply by brianpetro** [reply]
Just 1 more!

**Reply by brianpetro** [reply]
I added a link to join the discord server to the above post, since the provided link seems to not work on mobile and maybe other situations.

**Reply by fabioscarsi** [reply]
I already have access to Version 2. So, no screenshot. But I voted anyway for Smart Connections.

**Comment by franciscoabenza** [comment]
2nd 🥈
